docker/Dockerfile

```
@@ -115,6 +115,9 @@ FROM base AS risingwave
 
 LABEL org.opencontainers.image.source https://github.com/risingwavelabs/risingwave
 
+RUN apt-get update && apt-get -y install linux-tools-generic \
+    && ln -s "$(find /usr/lib/linux-tools/*/perf | head -1)" /usr/local/bin/perf
```

nit: no need to `apt-update` any more as we did it at the very beginning.
```suggestion
RUN apt-get -y install linux-tools-generic \
    && ln -s "$(find /usr/lib/linux-tools/*/perf | head -1)" /usr/local/bin/perf
```

---

I thought so at first, but it doesn’t seem to work.
https://buildkite.com/risingwavelabs/docker/builds/14698#018bfaf6-f601-4056-8f8c-f22186a8d5b7